src/Entity/ParagraphsRowType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\react_paragraphs\Entity;
+
+use Drupal\paragraphs\Entity\ParagraphsType;
+
+/**
+ * Defines the Paragraphs Row type entity.
+ *
+ * @ConfigEntityType(
+ *   id = "paragraphs_row_type",
+ *   label = @Translation("Row type"),
+ *   handlers = {
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
+ *     "list_builder" = "Drupal\react_paragraphs\ParagraphsRowTypeListBuilder",
+ *     "form" = {
+ *       "add" = "Drupal\react_paragraphs\Form\ParagraphsRowTypeForm",
+ *       "edit" = "Drupal\react_paragraphs\Form\ParagraphsRowTypeForm",
+ *       "delete" = "Drupal\react_paragraphs\Form\ParagraphsRowTypeDeleteForm"
+ *     },
+ *     "route_provider" = {
+ *       "html" = "Drupal\react_paragraphs\ParagraphsRowTypeHtmlRouteProvider",
+ *     },
+ *   },
+ *   config_prefix = "paragraphs_row_type",
+ *   admin_permission = "administer site configuration",
```

Comment:
Perhaps this should be its own permission

Reply:
```suggestion
 *   admin_permission = "administer paragraphs row entities",
```

Reply:
indeed. i'll need to add that to the permissions yml too.